[merged] Make upgrade version a positional argument.

mbarnes opened the pull request:
I should have just done this with the module work, but the `--upgrade_to` option which is not optional is proving to be an awkward wart in the manpage I'm writing for `atomic`.  I'd prefer it be positional: `commctl create upgrade my_cluster 1.2.3`

Also, somewhat related, but are you still okay with `create` as a verb for `upgrade` and `restart`?  I think we once talked about it sounding a little awkward, as opposed to something like `start`.  Once I add this stuff to `atomic`, I feel like we're kinda stuck with it.  So last call for changes.

mbarnes committed:
Make upgrade version a positional argument.

ashcrow commented:
Looking over this now.

ashcrow commented:
:+1: to the change.

I'll happily go with `start` for `upgrade` and `restart`. If you want to include it in this PR I'll wait to merge,

mbarnes commented:
Okay, working on a `start` command...

mbarnes committed:
Add a 'start' command for asynchronous operations.

It reads a little better than 'create', I think.

e.g.  commctl create upgrade my_cluster 7.3.0

      - vs -

      commctl start upgrade my_cluster 7.3.0

mbarnes commented:
:arrow_up: Start finished.

ashcrow commented:
LGTM

@cgwalters-bot try

ashcrow commented:
@cgwalters-bot r+
